docs/tutorial/plot_bbegin_measure_time.py

```diff
@@ -32,7 +32,10 @@
 from onnxruntime import InferenceSession
 from skl2onnx import to_onnx
 from skl2onnx.tutorial import measure_time
-
+import matplotlib
+# comment below line to show graph 
+matplotlib.use('Agg')
```

Comment:
I'm wondering if this change affects the rendering in the documentation?

Reply:
please let me know where the document was generated? I'm going to do the testing.

Reply:
You have to generate it yourself. The outcome is published in branch github-pages. Every modification in that branch is propagated into the website https://github.com/onnx/sklearn-onnx. You may use the following command line to build the documentation : 

```
python -m sphinx -j2 -v -N -T -b html -d build/doctrees docs dist/html
```